src/data/grv_perf_list.json

```diff
@@ -0,0 +1,236 @@
+[
+  {
+    "name": "ipc",
+    "nrs": [
+      {
+        "perf_type": "RAW",
+        "name": "Instructions",
+        "config": 8
+      }
+    ],
+    "drs": [
+      {
+        "perf_type": "RAW",
+        "name": "Cycles",
+        "config": 17
```

Comment:
I just realized that all these numbers are in decimal because JSON doesn't support hex integers directly.

Unfortunately pretty much every PMU document ever will list counters in hex.  This makes inspecting or editing these counters very painful because anything over than 0x20 becomes difficult to convert in my head.

What if we make config a string and then instead of `"config": 17` we could write `"config": "0x11"`.

Reply:
Yeah. That's an improvement we can consider.